jobs/hyperkube_release.groovy

```diff
@@ -0,0 +1,33 @@
+// META
+repo = "kubernetes-incubator/bootkube"
+
+// CONFIG
+org_whitelist = ['coreos', 'coreos-inc']
+job_admins = ['colemickens', 'ericchiang', 'rithujohn191', 'rphillips']
+user_whitelist = job_admins
+
+// JOBS
+job_name = "tku-2-release-hyperkube"
+
+pipelineJob(job_name) {
+  parameters {
+    stringParam('BOOTKUBE_VERSION', 'origin/master', 'git reference to build')
```

Comment:
Do we want to default to `origin/master` here? I assume we normally only want to release specific tags. I almost think we want to default to `""` and then error if the user didn't set a revision. Same for KUBERNETES_VERSION potentially?

Reply:
BOOTKUBE_VERSION in this case is the branch to use to find the Jenkinsfile. I am using it for testing changes to the Jenkins scripts. perhaps the description should be updated?

Reply:
Oh, my mistake, maybe a different name would be more intuitive, but it's fine either way.

Reply:
fixed this up... uses JENKINS_BRANCH